packages/ds-healthcare-gov/src/components/Footer/Footer.jsx

```diff
@@ -49,6 +50,10 @@ _Footer.propTypes = {
    * `https://healthcare.gov/topics`.
    */
   primaryDomain: PropTypes.string,
+  /**
+   * Element to be added to display content for top footer section changes
+   */
+  footerTop: PropTypes.node,
```

Comment:
We've got some intermediate Typescript typing going on right now, so any time a prop is added, we also have to update the type definition file. Could you please update [this file](https://github.com/CMSgov/design-system/blob/ff52e04552042f02931fc93359325b93616e4f1d/packages/ds-healthcare-gov/src/types/Footer/index.d.ts#L6) with

```
  footerTop?: React.ReactNode;
```
in the `FooterProps` interface

Reply:
> 

@scheul93 , After adding `footerTop?: React.ReactNode;` line, in `index.d.ts`, I am getting Enlint error. Which is not letting me commit my changes. Any ideas?

```
✖ eslint found some errors. Please fix them and try committing again.
/Users/niralkunadia/Localdev/cms/design-system/packages/ds-healthcare-gov/src/types/Footer/index.d.ts
11:24  error  Parsing error: Unexpected token, expected "{"
 9 | 
10 | export class Footer extends React.Component<FooterProps, any> {
> 11 |   render(): JSX.Element;
   |                        ^
12 | }
```

Reply:
@scheul93 , With this changes, Do I need to update snapshot? I am getting eslint error, for adding this line `footerTop?: React.ReactNode;`

Reply:
Added `footerTop?: React.ReactNode;` to index.d.ts